lib/Service/ApiService.php

```diff
@@ -127,19 +129,26 @@ public function create($fileId = null, $filePath = null, $token = null, $guestNa
 		}
 
 		$session = $this->sessionService->initSession($document->getId(), $guestName);
+
 		try {
 			$baseFile = $this->documentService->getBaseFile($document->getId());
 			$content = $baseFile->getContent();
-
-			$content = $this->encodingService->encodeToUtf8($content);
```

Comment:
Removed by accident?

Reply:
* removed on purpose when dropping the BaseFile
* not brought back when bringing it back
* remove BaseFile again as i am now confident we don't need it

So should be fine now. It does not apply to the state file because that contains base64 encoded content and that should not have utf8 content.